### PR TITLE
[automation] Auto-update linters version, help and documentation

### PR DESCRIPTION
**Pull request by nvuillam**
[automation] Auto-update linters version, help and documentation

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ FROM yoheimuta/protolint:latest as protolint
 FROM golang:alpine as dustilock
 RUN GOBIN=/usr/bin go install github.com/checkmarx/dustilock@v1.2.0
 
-FROM zricethezav/gitleaks:v8.18.1 as gitleaks
+FROM zricethezav/gitleaks:v8.18.0 as gitleaks
```
Comment (echoix) [inline Dockerfile:37]:
```suggestion
FROM zricethezav/gitleaks:v8.18.1 as gitleaks
```

**Comment by echoix** [comment]
@nvuillam you'll need to override and merge this so that the c_cland_format.md docs gets included so it could not fail. Unless you have another idea

**Comment by nvuillam** [comment]
@echoix done :)

**Pull request merged**